src/components/TracePage/TraceTimelineViewer/SpanDetail/AccordianKeyValues.js

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import KeyValuesTable from './KeyValuesTable';
+import toggleEnhancer from './toggle-enhancer';
+
+import './AccordianKeyValues.css';
+
+const dataPropType = PropTypes.arrayOf(
```

Comment:
Adding flow typing, which is handy for props, definitely reminded me why I'm skeptical — bugs.

- [Changes](https://github.com/uber/jaeger-ui/pull/64/commits/5c7f896ed4b39836af91b81995bd0b18f0cf0f60#diff-cbab2020bb1863553129838da2e79e95) that gave me trouble
- [Underlying issue](https://github.com/facebook/flow/issues/1511)
- [Simplified example](https://goo.gl/4Z23MJ)